### ISSUE TITLE
Demo this repo?

### ISSUE THREAD

**Issue by SaileshBechar** [openclimatefix/metnet#4]
Hello team at Open Climate Fix,

I am really interested in the code you developed here in this repo! Is there an easy way to demo this code with real images from GOES and MRMS? I have the described data but am not sure how to interface with the code base.

Thanks again,
Sailesh

**Comment by jacobbieker** [comment]
Hi, it should be fairly easy to get this working with those datasets. All that needs to be done would be to concatentate the GOES and MRMS image together, and then put them through the model. An example is sorta shown in the tests, in `tests/test_model.py` the only thing you would need tochange possibly is number of input and output channels. We are working on adding this to HuggingFace and making easy demos with Gradio, but haven't had the time yet to do so!

**Comment by SaileshBechar** [comment]
Great thank you Jacob! If I don't have to MRMS images to concatenate I can just change the dimensions of the input images correct? Sorry, I am new to all these neural networks. In addition, would it be okay to use some of this code in my 4th year design project? (with proper reference of course) We are building an app that would leverage the MetNet model

**Comment by jacobbieker** [comment]
Yeah! Just change the input channels and output channels and it should just work! And of course, use it however you want! We'd love to see how other people use these models we are implementing

**Comment by peterdudfield** [comment]
@all-contributors please add @SaileshBechar  for question
